site/content/docs/install_aio.md

```diff
@@ -0,0 +1,451 @@
+MYSQL\_ROOT\_PASSWORD | password | MySQL root user password.  The default is **OpenBMP**.  The root password can be changed using [standard MySQL instructions](https://dev.mysql.com/doc/refman/5.6/en/resetting-permissions.html).  If you do change the password, you will need to run the container with this env set.
+MYSQL\_OPENBMP\_PASSWORD | password | MySQL openbmp user password.  The default is **openbmp**.  You can change the default openbmp user password using [standard mysql instructions](https://dev.mysql.com/doc/refman/5.6/en/set-password.html).  If you change the openbmp user password you MUST use this env.  
+
+#### Run normally
+> ##### IMPORTANT
+> Make sure to define **API_FQDN** as a hostname (or fqdn) and not by IP.  The hostname should
+> resolve to the docker host (*host that runs docker containers*) IP address, which is normally
+> eth0.
+> 
+> If you do not plan to connect to the docker container via Kafka consumers, then you can use any 
+> hostname, such as *openbmp.localdomain*
+
+    docker run -d -e API_FQDN=<hostname> --name=openbmp_aio -e MEM=15 \
+         -v /var/openbmp/mysql:/data/mysql \
+         -p 3306:3306 -p 2181:2181 -p 9092:9092 -p 5000:5000 -p 8001:8001 \
+         openbmp/aio
+
+> ### Allow at least a few minutes for mysql to init the DB on first start
+
+
+
+### Monitoring/Troubleshooting
+Once the container is running you can run a **HTTP GET http://docker_host:8001/db_rest/v1/routers** to test that the API interface is working. 
+
+You can use standard docker exec commands to monitor the log files.  To monitor 
+openbmp, use ```docker exec openbmp_aio tail -f /var/log/openbmpd.log```
+
+Alternatively, it can be easier at times to navigate all the log files from within the container. You can do so using:
+    
+    docker exec -it openbmp_aio bash
+
+#### System Start/Restart Config (ubuntu 14.04)
+By default, the containers will not start automatically on system boot/startup.  You can use the below example to instruct the openbmp/aio container to start automatically. 
+
+You can read more at [Docker Host Integration](https://docs.docker.com/articles/host_integration/) on how to start containers automatically. 
+
+> #### IMPORTANT
+> The ```--name=openbmp_aio``` parameter given to the ```docker run``` command is used with the ```-a openbmp_aio``` parameter below to start the container by name instead of container ID.  You can use whatever name you want, but make sure to use the same name used in docker run.
+
+    cat <<END > /etc/init/aio-openbmp.conf
+    description "OpenBMP All-In-One container"
+    author "tim@openbmp.org"
+    start on filesystem and started docker
+    stop on runlevel [!2345]
+    respawn
+    script
+      /usr/bin/docker start -a openbmp_aio
+    end script
+    END
+     
+- - -     
+  
+
+Install on Ubuntu 14.04
+-----------------------
```

Comment:
We can keep this as is for now, but this pretty much needs to be rewritten as most of this has changed.